.gitignore

```diff
@@ -23,3 +23,5 @@ npm-debug.log*
 yarn-debug.log*
 yarn-error.log*
 lerna-debug.log*
+
+jest-cache/
```

Comment:
Let's keep this only in the `.gitignore` files of the individual packages.

Reply:
I had a jest-cache/ folder in my root folder for some reason. Git kept asking me to add it so i gitignored it.

Reply:
Sounds like that folder might have been stale? We don't seem to be running jest from the root so not sure how that could happen currently